solidity/contracts/TransformerRegistry.sol

```diff
@@ -111,6 +111,17 @@ contract TransformerRegistry is BaseTransformer, ITransformerRegistry {
     return abi.decode(_result, (uint256));
   }
 
+  /// @inheritdoc ITransformerRegistry
+  function transformAllToUnderlying(address _dependent, address _recipient) external returns (UnderlyingAmount[] memory) {
+    ITransformer _transformer = _getTransformerOrFail(_dependent);
+    uint256 _amountDependent = IERC20(_dependent).balanceOf(msg.sender);
+    bytes memory _result = _delegateToTransformer(
+      _transformer,
+      abi.encodeWithSelector(_transformer.transformToUnderlying.selector, _dependent, _amountDependent, _recipient)
+    );
+    return abi.decode(_result, (UnderlyingAmount[]));
+  }
```

Comment:
Knowing you, I'm amazed you didn't create an internal for this haha. Either way if you want to go with an internal call, or like this I'm on board.

Reply:
Hahahahahahaha. I was going to, but I couldn't do the same on #36 because the array was `memory` on the `transformAll` and `calldata` on `transform`

And I didn't want to convert calldata to memory just for an internal. So for the sake of consistency, I also didn't add an internal here